app/src/main/java/org/coepi/android/ui/alerts/AlertsViewModel.kt

```diff
@@ -30,33 +46,46 @@ class AlertsViewModel(
         .observeOn(mainThread())
         .toLiveData()
 
-    val title: LiveData<String> = alertsRepo.alerts
-        .map { title(it.size) }
-        .startWith(title(0))
-        .observeOn(mainThread())
-        .toLiveData()
-
     val updateStatusText: LiveData<String> = alertsRepo.updateReportsState
         .map { toUpdateStatusText(it) }
         .observeOn(mainThread())
         .toLiveData()
 
-    private fun title(alertsSize: Int) =
-        resources.getQuantityString(alerts_new_notifications_count, alertsSize)
-
     private fun Alert.toViewData(): AlertViewData =
         AlertViewData(
             exposureType = report.toExposuresString(),
-            contactTime = contactTime.toDate().toString(), // TODO which date format?
+            contactTime = contactTime.toTimeString(),
             report = this
         )
 
-    private fun PublicReport.toExposuresString(): String =
-        "Fever: $feverSeverity, cough: $coughSeverity, breathlessness: $breathlessness, " +
-                "earliestSymptomTime: $earliestSymptomTime"
+    @SuppressLint("SimpleDateFormat")
+    private fun UnixTime.toTimeString(): String {
+        val sdf = SimpleDateFormat("h:mm a")
+        return sdf.format(toDate())
+    }
 
-    fun onAlertAckClick(alert: AlertViewData) {
-        alertsRepo.removeAlert(alert.report)
+    @SuppressLint("DefaultLocale")
+    private fun PublicReport.toExposuresString(): String {
```

Comment:
Let's split this a bit more clearly: Add individual extensions to map cough severity etc. to strings. Call those here, e.g: `listOf(coughSeverity.toLocalizedString(),...).join("\n")`